gen/src/rfcgen.act

```diff
@@ -68,6 +68,24 @@ class SubStmt(object):
         argname = "None"
         return "<SubStmt name: %s  section: %s  argument_name: %s  cardinality: %s>" % (self.name, self.section, argname, self.cardinality)
 
+class MyAttr:
```

Comment:
Surely we can have a better name than MyAttr  ? the "My" doesn't really explain anything about it, nor does it have a description.

OrderedAttr?